pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/creator/impl/fwd/SingleValueFixedByteRawIndexCreator.java

```diff
@@ -30,7 +31,7 @@
 
 /**
  * Forward index creator for raw (non-dictionary-encoded) single-value column of fixed length data type (INT, LONG,
- * FLOAT, DOUBLE).
+ * FLOAT, DOUBLE, fixed-size BIG_DECIMAL).
```

Comment:
Suggest not adding this support now. Conceptually `STRING` and `BYTES` can also be fixed-size, so we should add this support separately to cover all of them. Currently we don't track whether the values are fixed size

Reply:
yeah .. will remove this support for now and we can add it later.